_pathways/path-mydb-mariadb.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Configure and Use a MariaDB Database on DB4Sci
+main_author: Michael Gutteridge
+primary_reviewers: atombaby, dtenenba, fizwit
+---
+
+This pathway will show you how to configure a MariaDB (MySQL) database on DB4Sci (aka MyDB) and connect to it using the command line tools on a rhino/gizmo node.
+
+## Pre-requisites
+A workstation, HutchNet ID, access to the internet and campus network, and browser
+
+If you are unfamiliar with any of these terms, hover over them to find more information.
+ - {% glossary rhino %}
+ - {% glossary gizmo %}
+
+## Steps
+
+### Log in to db4sci
+
+Browse to https://mydb.fredhutch.org/login and enter your HutchNet ID
+
+### Create the Database Container
+
+Once logged into db4sci, click over to "Create DB Container" and select "MariaDB (MySQL)" from the list.  This will present a form with a number of fields you must fill in:
+
```

Comment:
Most everything (lines 26-35) in this section should probably be moved to a new doc on creating databases in mydb and linked to from this section.